torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1384,13 +1379,18 @@ def call_function(
             args, kwargs
         )
 
-        p_args, p_kwargs, example_value = self.create_wrapped_node(tx, query, score_mod)
+        p_args, p_kwargs = self.create_wrapped_node(tx, query, score_mod)
         proxied_args = [query, key, value, *other_buffers]
 
         # Store the invocation as a call
         # Norm_kwargs contains the score_function and we dont want to proxy this because
```

Comment:
update the comment, the problem was that I copied pasta from the rest of the HOP implementations. However unlike the others, the subgrpah's output for this hop is unrelated to what the overall HOP returns, and thus needed to create the right output proxy.  I am not sure why this wasn't erroing before as the shapes for the output proxies were wonky.